joladnijo/settings.py

```diff
@@ -76,9 +77,12 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.mysql',
-        'NAME': 'joladnijo',
-        'USER': 'joladnijo',
+        'ENGINE': os.environ.get('DBENGINE', ''),
+        'NAME': os.environ.get('DBNAME', ''),
+        'USER': os.environ.get('DBUSER', ''),
+        'PASSWORD': os.environ.get('DBPASSWORD', ''),
+        'HOST': os.environ.get('DBHOST', ''),
```

Comment:
Default ertek itt lehet a localhost, mert akkor helyben futtatva az applikaciot nem kell kulon env.var-t beallitani.
```suggestion
        'HOST': os.environ.get('DBHOST', 'localhost'),
```